config.json

```diff
@@ -20,87 +20,258 @@
       ]
     },
     {
-      "slug": "leap",
-      "uuid": "b6acda85-5f62-4d9c-bb4f-42b7a360355a",
+      "slug": "two-fer",
+      "uuid": "4177de10-f767-4306-b45d-5e9c08ef4753",
       "core": true,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-        "booleans",
         "conditionals",
-        "logic"
+        "optional_values",
+        "text_formatting"
       ]
     },
     {
-      "slug": "reverse-string",
-      "uuid": "d39f86fe-db56-461c-8a93-d87058af8366",
-      "core": false,
+      "slug": "acronym",
+      "uuid": "038c7f7f-02f6-496f-9e16-9372621cc4cd",
+      "core": true,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
+        "regular_expressions",
         "strings"
       ]
     },
     {
-      "slug": "isogram",
-      "uuid": "d1a98c79-d3cc-4035-baab-0e334d2b6a57",
-      "core": false,
-      "unlocked_by": "two-fer",
+      "uuid": "574d6323-5ff5-4019-9ebe-0067daafba13",
+      "slug": "high-scores",
+      "core": true,
+      "unlocked_by": null,
       "difficulty": 1,
+      "topics": [
+        "control-flow (if-else statements)",
+        "sequences",
+        "text_formatting"
+      ]
+    },
+    {
+      "slug": "matrix",
+      "uuid": "b564927a-f08f-4287-9e8d-9bd5daa7081f",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+        "integers",
+        "loops",
+        "matrices",
+        "type_conversion"
+      ]
+    },
+    {
+      "slug": "word-count",
+      "uuid": "04316811-0bc3-4377-8ff5-5a300ba41d61",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 3,
+      "topics": [
+        "algorithms",
+        "logic",
+        "pattern_recognition",
+        "strings",
+        "text_formatting"
+      ]
+    },
+    {
+      "slug": "hamming",
+      "uuid": "8648fa0c-d85f-471b-a3ae-0f8c05222c89",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 2,
       "topics": [
         "algorithms",
         "conditionals",
+        "filtering",
+        "logic",
         "loops",
+        "sequences",
+        "sets",
         "strings"
       ]
     },
     {
-      "slug": "pangram",
-      "uuid": "bebf7ae6-1c35-48bc-926b-e053a975eb10",
-      "core": false,
-      "unlocked_by": "hello-world",
-      "difficulty": 1,
+      "slug": "raindrops",
+      "uuid": "82d82e32-cb30-4119-8862-d019563dd1e3",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": null,
+      "deprecated": false
```

Comment:
`deprecated` has is assumed `false` if not given, so it can be left out.

```suggestion
```

As for reinstating `raindrops`, I have no problem with this. I tracked down when it was deprecated, and since it was a commit made directly to master :angry: (bf9b7804a67dce1bbea263ec2fe5cb1fb4af7141), whatever discussion may have happened to decide that it should be deprecated is totally unlinked from the change; I rather liked that exercise myself.

Reply:
@iHiD 
> probably conditionally having an undeprecate method 

Could use just use the explicit `"deprecated": false`? Most exercises leave it out, so then it becomes:

- If `deprecated` not given, assume not previously or currently deprecated
- If `deprecated: true`, deprecate it
- If `deprecated: false`, undeprecate it

Reply:
I think the config pattern is fine (true if true else missing) but we just need some code in the website to check for things going from deprecated to not-deprecated as that's just not a pattern we've implemented on the parser.